script.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+input_file="web_server.log"
+error_200=$(grep -c 'HTTP/1.1" 200' "$input_file")
+cat <<EOF >>output_file
+ "Number of response code 200 : "$error_200"
+EOF
+
+declare -A codes
+
+for ((num=100; num<=999; num++)); do
```

Comment:
what if status code will be less than 100?? your script can't be able to find them

Reply:
Response codes starts from 100 so thats why kept the condition.